.github/workflows/review.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: v1.19
+          go-version: 1.19
```

Comment:
Noticed that our Go version names were inconsistent about using the `v`, and it caused me some confusion when managing required actions for merging PR, so just cleaned that up.